packages/replay/tsconfig.worker.json

```diff
@@ -7,7 +7,13 @@
     "esModuleInterop": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "target": "es5"
+    "target": "es5",
```

Comment:
l: Is it on purpose that we keep `es5` as target here? Just making sure :D

Reply:
Good point, we should probably change it to ES6